docs/block-client-apis/notices.md

```diff
@@ -0,0 +1,174 @@
+
+| Argument      | Type   | Description                                                                                                                                                                              |
+| ------------- | ------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `text`        | string | Text to be displayed in the notice.                                                                                                                                                      |
+| `noticeProps` | Object | Object with the notice props. Refer to the [Gutenberg docs](https://github.com/WordPress/gutenberg/blob/master/packages/notices/src/store/actions.js#L46) to know the available options. |
+
+#### `addSnackbarNotice( text = '', noticeProps = {} )`
+
+Create a new snackbar notice. If the context is not specified in the `options` object, the current context is used.
+
+| Argument      | Type   | Description                                                                                                                                                                              |
+| ------------- | ------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `text`        | string | Text to be displayed in the notice.                                                                                                                                                      |
+| `noticeProps` | Object | Object with the notice props. Refer to the [Gutenberg docs](https://github.com/WordPress/gutenberg/blob/master/packages/notices/src/store/actions.js#L46) to know the available options. |
+
+#### `hasNoticesOfType( type )`
+
+Checks whether there are notices of the provided type in the current context.
+
+| Argument | Type   | Description                   |
+| -------- | ------ | ----------------------------- |
+| `type`   | string | Type of the notices to check. |
+
+#### `notices`
+
+An array of the notices in the current context.
+
+#### `removeNotice( id, ctx )`
+
+Remove an existing notice. If the context is not specified, the current context is used.
+
+| Argument | Type   | Description                                                                                                 |
+| -------- | ------ | ----------------------------------------------------------------------------------------------------------- |
+| `id`     | string | Id of the notice to remove.                                                                                 |
+| `ctx`    | string | Context where the notice to remove is stored. If the context is not specified, the current context is used. |
+
+#### `removeNotices( status = null )`
```

Comment:
I know this is just a doc and this is not related but I feel like the API is all over the place when it comes to context, sometimes in an object, sometimes a string param and sometimes it's implied and you can't even set it.

Reply:
Yup, in general it uses the current context so it's implied, but some specific methods allow specifying a different context. That's useful for payment method processing, for example, so they can add errors either to the express payment methods or the regular ones. I think we could iterate to make the API more clear after the first release.